### PR TITLE
feat: Overview Page adding the overview page and page ref for the navigation - MEED-550 - Meeds-io/MIPs#10

### PR DESCRIPTION
**Pull request by ArfaouiMohamedAmine**
this change is going to add overview page and its reference to the navigation

### CODE REVIEW

File: webapps/plf-root-webapp/src/main/webapp/WEB-INF/conf/portal/global/navigation.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 Meeds Association
```
Comment (plamarque) [inline webapps/plf-root-webapp/src/main/webapp/WEB-INF/conf/portal/global/navigation.xml]:
```suggestion
Copyright (C) 2022 Meeds Association
```

File: webapps/plf-root-webapp/src/main/webapp/WEB-INF/conf/portal/global/pages.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 Meeds Association
```
Comment (plamarque) [inline webapps/plf-root-webapp/src/main/webapp/WEB-INF/conf/portal/global/pages.xml]:
```suggestion
Copyright (C) 2022 Meeds Association
```

File: webapps/plf-root-webapp/src/main/webapp/WEB-INF/conf/portal/global/pages.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+<page-set xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.exoplatform.org/xml/ns/gatein_objects_1_8 http://www.exoplatform.org/xml/ns/gatein_objects_1_8"
+  xmlns="http://www.exoplatform.org/xml/ns/gatein_objects_1_8">
+
+  <page>
+    <name>overview</name>
+    <title>overview</title>
+    <access-permissions>*:/platform/users;*:/platform/externals</access-permissions>
+    <edit-permission>*:/platform/rewarding</edit-permission>
```
Comment (plamarque) [inline webapps/plf-root-webapp/src/main/webapp/WEB-INF/conf/portal/global/pages.xml]: isn't *:/platform/administrators  mode appropriate here?

Comment (ArfaouiMohamedAmine) [reply webapps/plf-root-webapp/src/main/webapp/WEB-INF/conf/portal/global/pages.xml]: The '*:/platform/rewarding' was mentioned in the technical spec 
Is there a need for all platform administrator that I should be aware of ? @plamarque

Comment (boubaker) [reply webapps/plf-root-webapp/src/main/webapp/WEB-INF/conf/portal/global/pages.xml]: Right @ArfaouiMohamedAmine ! We will need here to make it editable by `/platform/rewarding` group in order to let members of this group to change portlet preferences through settings button.

**Review by AzmiTouil — APPROVED** [review]

**Pull request merged**